utils/vlog/vlog_to_model.py

```diff
@@ -0,0 +1,92 @@
+#!/usr/bin/python3
+import yosys_exec
+import lxml.etree as ET
+import argparse, re
+import os, tempfile
+from yosys_json import YosysJson
+
+"""
+Convert a Verilog simulation model to a VPR `model.xml`
+"""
+
+parser = argparse.ArgumentParser(description='Convert a Verilog simulation model into a VPR model.xml file')
+parser.add_argument('infiles', metavar='input.v', type=str, nargs='+',
+                    help='one or more Verilog input files')
+parser.add_argument('--top', help='top level module, not needed if only one module across all files')
+parser.add_argument('-o', help='output filename, default model.xml')
+
+args = parser.parse_args()
+
+aig_json = yosys_exec.vlog_to_json(args.infiles, True, True)
```

Comment:
I'd go with `yosys.verilog_to_json` personally.

Reply:
Yes, that makes sense. Will put Yosys stuff into a new folder